src/views/compute/DeployVM.vue

```diff
@@ -1791,4 +1806,10 @@ export default {
   .form-item-hidden {
     display: none;
   }
+
+  .vapp-category {
+    background-color: lightblue;
```

Comment:
![Screenshot from 2020-08-19 15-04-24](https://user-images.githubusercontent.com/153340/90618225-4f4be100-e22d-11ea-82af-8af90da750ab.png)
Can we use some standard primate color. Looks different to rest of the UI

Reply:
use the alert component